### DISCUSSION TITLE
How do I launch an image picker with options?

### DISCUSSION THREAD

**Discussion by moonman239** [react-native-image-picker/react-native-image-picker#1753] [Q&A]
I'm trying to launch an image picker using launchImagePicker, but I need to enable getting the base 64 strings of images. However, I'm not sure how to add the option to the "options" parameter. How do I do this?

**Comment by Johan-dutoit** [comment]
`launchImagePicker` doesn't exist since v3.0.0. That being said, `launchImageLibrary` and `launchCamera` both accept options i.e. `launchCamera(options)`
